src/ui/views/2_quantity/index.js

```diff
@@ -23,12 +23,17 @@ export class QuantityView extends React.Component {
       quantity: this.state.quantity
     })
   }
-
+  
+  changeViewProducts = () => {
```

Comment:
can you change this name to `goToProducts`?

and can you change the other method from `changeView` to `goToAssignedOffer` ?